integration-tests/tests/src/index.ts

```diff
@@ -22,8 +22,8 @@ import { flags } from "realm";
 
 // TODO: Refactor tests to disable this
 flags.ALLOW_VALUES_ARRAYS = true;
-// We need this to call Realm.cleanTestState()
-flags.CLEAN_TEST_STATE = true;
+// We need this to call Realm.clearTestState()
+flags.CLEAR_TEST_STATE = true;
```

Comment:
```suggestion
flags.ENABLE_CLEAR_TEST_STATE = true;
```

Or `ALLOW`. The point is that setting the flag doesn't by itself clear the test state, it mearly *enables* (or allows) the test state to be cleared.

Also, reading this now, I think `clearTestState()` is slightly misnamed. Really it should be something like `clearStateAfterTest()` because it doesn't clear the _test_ state, it tries to reset the general state of the world. But I don't suggest changing that right now.

Reply:
I opted for `ALLOW` to align with the other existing flag.

And I agree, the function name is confusing. It's internal and can easily be renamed in another PR 👍